website/docs/getting-started/setup/instance-configuration/authentication/security-assertion-markup-language-saml/auth0.mdx

```diff
@@ -0,0 +1,82 @@
+
+## Prerequisites
+
+1. In Appsmith, go to **Admin Settings** > **Authentication** and click **Enable** on  **SAML 2.0**.
+
+2. Copy the **Redirect URL** and **Entity URL** from the **SAML 2.0** configuration page to add them later in the Auth0 settings. 
+
+##  Configure Auth0
+
+1. Log in to your [Auth0](https://auth0.com/) account and go to **Applications** > **Create Application**. 
+2. In the **Create application** modal, select **Regular Web Application** and click **Create**.
+3. Once your application is created, you’re taken to the Quick Start screen. Go to the **Settings** tab.
+4. Scroll down to the **Application URIs** and add the **Redirect URL** in the **Allowed Callback URLs** field.
+5. Click **Save Changes**.
+
+## Configure Auth0 in Appsmith
+
+To complete the SAML configuration, you’ll have to register the identity provider on Appsmith. Appsmith provides three options to register the identity provider as mentioned below:
+
+<Tabs queryString="auth0-saml">
+<TabItem label="Metadata URL (recommended)" value="auth0metadataurl">
+
+To get the Metadata URL for your Auth0 application, follow the steps below:
+1. Go to **Settings** tab in your Auth0 application, scroll down, and expand the **Advanced Settings** sections.
+2. Click the **Endpoints** tab, scroll down to the **SAML** section, and copy the SAML **Metadata URL**.
+3. In the Appsmith **SAML 2.0** configuration page, add the Metadata URL in the **Metadata URL** field under the **Register Identity Provider** section.
+
+</TabItem>
+
+<TabItem label="Metadata XML" value="auth0metdataxml">
+
+To set up SAML using the raw Metadata XML file, follow the steps below:
+1. Open the copied **Metadata URL** in a browser tab and copy the XML content.
+2. Navigate to Appsmith and add the raw XML in the **Metadata XML** field under the **Register Identity Provider** section in the **SAML 2.0** configuration page.
+
+</TabItem>
+<TabItem label="Idp data" value="auth0idpdata">
+
+If you have Identity provider data like **X509 Public Certificate**, **Email**, you can choose this option to configure SAML.
+
+1. Open the Metadata URL in a browser tab or open the Metadata XML file and locate the following XML tags:
+
+    a. `<EntityDescriptor>`
+
+    b. `<SingleSignOnService>`
+
+    c. `<X509Certificate>`
+    
+    d. `<NameIDFormat>`
+
+2. In the Appsmith **SAML 2.0** configuration page, click **IdP Data** under the **Register Identity Provider** section. Add the following values from XML tags as per the table below:
+
+| <div style= {{width:"200px"}}> **Appsmith Field ** </div>  | <div style= {{width:"auto"}}> **Metadata XML Tag**  </div>                                                                      |
+| ------------------- | ------------------------------------------------------------------------------------------ |
+| **Entity ID**               | Add the value of **entityID** attribute from `<EntityDescriptor>` tag.                 |
```

Comment:
```suggestion
| **Entity ID**               | Paste the value of the **entityID** attribute specified in the `<EntityDescriptor>` tag.                 |
```

Reply:
Sorry I committed it. But the change you suggested here changes the meaning of the sentence. You have to paste the value of the entityID attribute present in the `<EntityDescriptor>` tag to the Entity ID in Appsmith. I'm telling the user what to add in Entity ID.